mmpose/models/keypoint_heads/temporal_regression_head.py

```diff
@@ -41,9 +45,33 @@ def __init__(self,
         self.conv = build_conv_layer(
             dict(type='Conv1d'), in_channels, num_joints * 3, 1)
 
+        if self.max_norm is not None:
+            # Apply weight norm clip to conv layers
+            weight_clip = WeightNormClipRegister(self.max_norm)
+            for module in self.modules():
+                if isinstance(module, nn.modules.conv._ConvNd):
+                    weight_clip.register(module)
+
+    def _transform_inputs(self, x):
```

Comment:
Why not use the '_transform_inputs()' in `top_down_simple_head.py`?

Reply:
The behaviors of these two methods are slightly different. TemporalRegressionHead handles temporal features which may have different dimensions from image features'.